sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonBenchmarks.scala

```diff
@@ -25,8 +25,14 @@ import org.apache.spark.util.{Benchmark, Utils}
 
 /**
  * The benchmarks aims to measure performance of JSON parsing when encoding is set and isn't.
- * To run this:
- *  spark-submit --class <this class> --jars <spark sql test jar>
+ * To run:
+ *  mvn clean package -pl sql/core -DskipTests
+ *  ./dev/make-distribution.sh --name local-dist
+ *  cd dist/
+ *  ./bin/spark-submit bin/spark-submit \
+ *      --class org.apache.spark.sql.execution.datasources.json.JSONBenchmarks \
+ *      --jars ./sql/core/target/spark-sql_*-tests.jar
+ *  open /tmp/output.txt
```

Comment:
Do we need to update it like this? I prefer original version...

Reply:
yea.. similar comment at https://github.com/apache/spark/pull/21596#discussion_r197661126

Reply:
I've reverted this